pip_review/__main__.py

```diff
@@ -1,18 +1,15 @@
 from __future__ import absolute_import
-import os
 import re
 import argparse
 from functools import partial
 import logging
 import json
 import sys
 import pip
-import subprocess
 from packaging import version
 
 PY3 = sys.version_info.major == 3
 if PY3:  # Python3 Imports
-    import urllib.request as urllib_request
```

Comment:
You were right to remove this, but you forgot to remove it in the Python 2 branch on line 29 as well. I already fixed that for you.